src/mono/sample/iOS/Program.cs

```diff
@@ -37,7 +37,14 @@ private static void OnButtonClick()
     {
         SetText("OnButtonClick! #" + counter++);
     }
-
+#if CORECLR_TEST
```

Comment:
Why do we need the special case for CoreCLR? Cannot it run the same main as we use with Mono?